puree/src/main/java/com/cookpad/puree/PureeConfiguration.java

```diff
@@ -52,17 +54,23 @@ public PureeStorage getStorage() {
         return sourceOutputMap.get(logClass);
     }
 
+    public int getDeleteThresholdInRows() {
+        return deleteThresholdInRows;
```

Comment:
I think the parameter `deleteThreshldInRows` should not be the field of PureeLogger. 

Let `Puree#truncateBufferedLogs()` to take the number of rows to truncate and remove the field from this class. It is enough.